dashboard/test/models/user_test.rb

```diff
@@ -1563,6 +1563,18 @@ def track_progress(user_id, script_level, result, pairings: nil)
     assert_equal [], teacher.reload.permissions
   end
 
+  test 'assign_course_as_facilitator assigns course to facilitator' do
+    facilitator = create :facilitator
+    facilitator.course_as_facilitator = Pd::Workshop::COURSES.last
+    assert facilitator.courses_as_facilitator.where(course: Pd::Workshop::COURSES.last).any?
+  end
+
+  test 'delete_course_as_facilitator removes facilitator course' do
+    facilitator = create :facilitator
+    facilitator.delete_course_as_facilitator Pd::Workshop::COURSES.first
+    refute facilitator.courses_as_facilitator.where(course: Pd::Workshop::COURSES.first).any?
+  end
```

Comment:
I had already updated factories.rb to assign a course when creating a facilitator.  I temporarily added an assert before the delete to verify `facilitator` has that course and it does.

I was not able to get `assert_difference` to work for them same reason, I had trouble debugging `assert_create` in the assign test.  Leaving the test unchanged until I learn enough ruby to diagnose this.